spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -5,6 +5,13 @@
   include OidcAuthHelper
   include DocAuthHelper
 
+  it 'it sets the sp_issuer cookie' do
+    visit_idp_from_ial1_oidc_sp
+
+    cookie = cookies.filter { |c| c.name == 'sp_issuer' }.first.value
```

Comment:
`.filter {}.first` is the same as `.find {}` right?

Reply:
Yup. 🤦